packages/core/src/index.ts

```diff
@@ -821,4 +823,4 @@ function effect(fn: EffectFn): () => void {
 	return effect._dispose.bind(effect);
 }
 
-export { signal, computed, effect, batch, untracked, Signal, ReadonlySignal };
+export { computed, effect, batch, untracked, Signal, ReadonlySignal };
```

Comment:
Is this intended?

Reply:
Yes, overloads need to be exported alongside their definition 😅

Reply:
Code blindness, didn't even notice the `export` statements above. Sorry!